thunder/dynamo/utils.py

```diff
@@ -781,37 +760,122 @@ def torch_inductor(fn, inputs):
 
     fx_graph = symbolic_trace(fn)
     return inductor_compile(fx_graph, inputs)
+"""
+    EXECUTOR_DICT_CODE_STR = f"""{TORCH_INDUCTOR_FUNCTION_STR}
 
 bench_executors_dict = {{}}
 bench_executors_dict["thunder"]=partial(thunder.jit, {thunder_options_str})
 bench_executors_dict["torch_inductor"]=torch_inductor
 bench_executors_dict["eager"]=None
 """
-            if has_cuda_args:
-                code_str += f"""bench_executors_dict["thunder_cugraph"]=partial(thunder.jit, transform=CUDAGraphTransform())\n"""
-            code_str += f"""
+    if has_cuda_args:
+        EXECUTOR_DICT_CODE_STR = f"""{EXECUTOR_DICT_CODE_STR}
+bench_executors_dict["thunder_cugraph"]=partial(thunder.jit, transform=CUDAGraphTransform())
+"""
+
+    COMMAND_LINE_ARGS = f"""
```

Comment:
No need for f as there is no variable included in the string
```suggestion
    COMMAND_LINE_ARGS = """
```